assets/src/domain/eventEditor/ui/tickets/ticketPriceCalculator/inputs/styles.scss

```diff
@@ -0,0 +1,13 @@
+$input-number: 'input[type=' number ']';
+
+.ee-input {
+	&__price-field {
+		&--has-error {
+			// this was used to increase specificity, more info: https://sass-lang.com/documentation/at-rules/at-root
+			@at-root #{$input-number}#{&} {
```

Comment:
this selector will result in `input[type=number].ee-input__price-field--has-error`

Reply:
if I'd use here `&input[type=' number ']`, the result would be `.ee-input__price-field--has-errorinput[type=' number ']` which is invalid selector

Reply:
cool... did not know about the `@at-root` rule :+

Reply:
neither did I, google enlightened me